### ISSUE TITLE
Move custom CSS and JS over to this repository

### ISSUE THREAD

**Issue by jakub-pomykala** [OpenLiberty/quick-labs#315]
For the previous time the custom CSS and the JS have been stored in Daniels personal repo. However we should move this over to this repo and also consider moving over the whole eDX theme and store it here.

**Comment by jakub-pomykala** [comment]
The custom CSS and JS has now been merged into prod in a new dir called "Portal-Assets". Just waiting on switching over the eDX to get the source code from here instead of Daniel's repo

**Comment by jakub-pomykala** [comment]
Closing issue as we have moved this over inside the LMS.zip
